config/metac.yaml

```diff
@@ -64,4 +64,54 @@ spec:
   hooks:
     sync:
       inline:
-        funcName: sync/command
\ No newline at end of file
+        funcName: sync/command
+---
+apiVersion: dope/v1
+kind: GenericController
+metadata:
+  name: finalize-command
+  namespace: dope
+spec:
+  watch:
+    apiVersion: dope.mayadata.io/v1
+    resource: commands
+  attachments:
+  # Delete pod
+  - apiVersion: v1
+    resource: pods
+    advancedSelector:
+      selectorTerms:
+      # select pod if its namespace is same as watch namespace
+      - matchFields:
+          metadata.namespace: EqualsWatchNamespace
```

Comment:
Can you please check if EqualsWatchNamespace is really supported for matchFields?
Please check what's the supported key & value as well.
https://github.com/AmitKumarDas/metac/blob/master/apis/metacontroller/v1alpha1/selector.go

Reply:
https://github.com/AmitKumarDas/metac/tree/master/controller/common/selector
https://github.com/AmitKumarDas/metac/blob/master/controller/common/selector/reference_test.go

Reply:
Not aware!! I looked at API and written this. There is no information written in APIs

Reply:
https://github.com/AmitKumarDas/metac/blob/master/apis/metacontroller/v1alpha1/selector.go -- API link

Reply:
Updated